android/fakewallet/build.gradle

```diff
@@ -52,7 +52,8 @@ dependencies {
     implementation 'androidx.navigation:navigation-fragment-ktx:2.4.2'
     implementation 'androidx.room:room-runtime:2.4.2'
     implementation 'androidx.room:room-ktx:2.4.2'
-    implementation 'com.github.bumptech.glide:glide:4.13.2'
+    implementation("io.coil-kt:coil:2.1.0")
```

Comment:
MINOR: can we keep this list sorted alphabetically? Helps when us poor humans avoid missing something when we need to understand a projects dependencies.